Formula/f/fedify.rb

```diff
@@ -0,0 +1,29 @@
+class Fedify < Formula
+  desc "CLI toolchain for Fedify"
+  homepage "https://fedify.dev/cli"
+  url "https://github.com/fedify-dev/fedify/archive/refs/tags/1.5.0.tar.gz"
+  sha256 "aaf39a8a5c9127d958fe261deb95985a6f7a76994f4209af9466a75b7a993303"
+  license "MIT"
+  head "https://github.com/fedify-dev/fedify.git", branch: "main"
+
+  depends_on "deno" => :build
+
+  def install
+    system "deno", "task", "codegen"
+    system "deno", "compile", "--allow-all", "--output=#{bin/"fedify"}", "cli/mod.ts"
+    generate_completions_from_executable(bin/"fedify", "completions")
+  end
+
+  test do
+    version = shell_output "#{bin}/fedify --version"
+    assert_equal "[1mfedify[22m [94m1.5.0[39m", version.strip
```

Comment:
```suggestion
    version_output = shell_output "#{bin}/fedify --version"
    assert_match version.to_s, version_output
```

Reply:
Can you check where fedify outputs it's version? From the failing test it looks like it goes to `stderr`.

Reply:
I double-checked it, and it goes to stdout… I believe the problem is that it immediately terminates right after invoked, without any outputs, even with zero exit code, on Linux CI. (It can't be reproduced on my Linux boxes, even inside the container with the same image with the Linux CI.)

Reply:
Did you try contacting upstream about it? It seems like a real bug to me

Reply:
I'm a maintainer of the upstream, and yeah, it could be a real bug… I suspect it's a bug of [`deno compile`](https://docs.deno.com/runtime/reference/cli/compile/).  Or does homebrew-core have any guideline to install a Deno package (i.e., [`deno install -g`](https://docs.deno.com/runtime/reference/cli/install/#global-installation))? If so I can rewrite the formula to avoid `deno compile` at all.

Reply:
Is there a way we can debug that, here in CI?

Reply:
I have no idea, but how about using `strace`?